apps/rush-lib/src/data/test/ChangeFile.test.ts

```diff
@@ -11,7 +11,7 @@ import { ChangeType } from '../ChangeManagement';
 
 describe('ChangeFile', () => {
   it('can add a change', () => {
-    const rushFilename: string = path.resolve(__dirname, 'repo', 'rush.json');
+    const rushFilename: string = path.resolve(__dirname, 'repo', 'rush-npm.json');
```

Comment:
>rush-npm [](start = 66, length = 8)

why do we have to rename it?

Reply:
We renamed it (only in these tests) to distinguish the NPM version of the rush.json from the PNPM version (which differ by some fields)

---
In reply to: [152134278](https://github.com/Microsoft/web-build-tools/pull/446#discussion_r152134278) [](ancestors = 152134278)